### PR TITLE
kevbe added readme message

### PR DESCRIPTION
**Pull request by egume**
i added a read me file

### CODE REVIEW

File: ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a read me file created by egume
```
Comment (Oghenekevbe) [inline ReadMe.txt]: is this all?

Comment (egume) [reply ReadMe.txt]: yes, should i add more?

Comment (Oghenekevbe) [reply ReadMe.txt]: add another line

**Review by Oghenekevbe — APPROVED** [review]
okay

**Pull request merged**